tests/flows/test_resume.py

```diff
@@ -36,3 +37,45 @@ def test_resume(gcd_chip):
     # Ensure flow finished successfully
     assert gcd_chip.find_result('def', step='place') is not None
     assert gcd_chip.find_result('gds', step='export') is not None
+
+@pytest.mark.eda
+@pytest.mark.timeout(600)
+def test_resume_remote(gcd_chip, unused_tcp_port):
+    '''Test that jobs can be resumed on a remote server.
+    '''
```

Comment:
move to remote tests

Reply:
Sure, I guess it makes more sense there.